src/generator/helpers.ts

```diff
@@ -11,7 +11,7 @@ import { ArraySchema, CodeModel, DictionarySchema, Language, Parameter, Schema,
 import { ImportManager } from './imports';
 
 export const dateFormat = '2006-01-02';
-export const datetimeRFC3339Format = 'time.RFC3339';
+export const datetimeRFC3339Format = 'time.RFC3339Nano';
```

Comment:
Just wondering, there no case where we need time.RFC3339?

Reply:
Or do the trailing zeros not matter?

Reply:
Omitting the trailing zeros just saves a few bytes, which in itself isn't a big deal, however we use Nano in other places so it's more about consistency.

Reply:
Ok, got it 👍